fcrepo4/fcrepo4
# Remove integration tests for HTTP MOVE and COPY

Resolves: https://jira.lyrasis.org/browse/FCREPO-3224

# What does this Pull Request do?
Removes integration tests for HTTP MOVE and COPY... and associated functional expectations.

# How should this be tested?
A successful build means success.
Also, ensure that no response headers contain any mention of COPY or MOVE

Note: [wiki documentation has also been updated](https://wiki.lyrasis.org/pages/diffpagesbyversion.action?pageId=178882392&selectedPageVersions=3&selectedPageVersions=2), removing mention of COPY/MOVE

# Interested parties
@fcrepo4/committers
